libs/components/src/lib/tag/tag.template.ts

```diff
@@ -32,8 +32,8 @@ const getClasses = ({
 function renderDismissButton(iconTag: string) {
 	return html<Tag>`
 	<span
-		aria-hidden="true"
 		class="dismiss-button"
+		aria-label="${(x) => `${x.label} - ${x.locale.tag.remove}`}"
```

Comment:
Since you changed the tag.remove to function accepting label as argument, this part should be changed to:
```suggestion
		aria-label="${(x) => x.locale.tag.remove(x.label) }"
```